app/keepers/keepers.go

```diff
@@ -571,13 +630,18 @@ func (ak *AppKeepers) InitKeepers(
 	transferStack = transfer.NewIBCModule(ak.TransferKeeper)
 	transferStack = ibcfee.NewIBCMiddleware(transferStack, ak.IBCFeeKeeper)
 
+	var zoneConciergeStack porttypes.IBCModule
+	zoneConciergeStack = zoneconcierge.NewIBCModule(ak.ZoneConciergeKeeper)
```

Comment:
```suggestion
	zoneConciergeStack := zoneconcierge.NewIBCModule(ak.ZoneConciergeKeeper)
```

Reply:
This seems necessary for L635. If I apply this then L635 will not compile